src/EnergyPlus/DataRuntimeLanguage.cc

```diff
@@ -177,15 +177,39 @@ namespace DataRuntimeLanguage {
     // Curve and Table access function
     int const FuncCurveValue(68);
 
-    int const NumPossibleOperators(68); // total number of operators and built-in functions
+    // Weather data query functions
+    int const FuncTodayIsRain(69);          // Access TodayIsRain(hour, timestep)
```

Comment:
New functions to access `Today*` and `Tomorrow*` weather data arrays. (And, yeah, an enum class would be great here, but didn't want to get into those weeds here.)